### ISSUE TITLE
client.time_series.get_time_series does not return metadata

### ISSUE THREAD

**Issue by senickel** [cognitedata/cognite-sdk-python#261]
**Describe the bug**
When executing `client.time_series.get_time_series()` with `include_metadata = True` no metadata is returned.

**To Reproduce**
Runnable code reproducing the error.
```
import cognite
import requests
import os
import numpy as np
import pandas as pd
from datetime import datetime, timedelta

from cognite.client.stable.time_series import TimeSeries
sm_api = os.environ['SM_API_KEY']
client = cognite.CogniteClient(api_key = sm_api)
ts_name = 'Test_tssssss'
my_time_series = [TimeSeries(name=ts_name, 
                             description = 'test_description',
                             metadata = { 'ASSETSCOPENAME' : 'meta_test_1' })]
client.time_series.post_time_series(my_time_series)

# create dummy data
np.random.seed(1338)
start_time = int((datetime.now()-timedelta(1)).strftime("%s"))
timestamps = [(start_time + i * 10)*1000 for i in np.arange(11)]
df = pd.DataFrame({'timestamp' : timestamps})
df[ts_name] = np.random.random(df.shape[0])
client.datapoints.post_datapoints_frame(df)
# get time_series
ts1 = client.time_series.get_time_series(name = ts_name,
                                         include_metadata = True).to_pandas()
ts1_id = ts1['id'].loc[0] 
print(ts1.loc[0])
# no meta data
# requests:
# first with no metadata
r1 = requests.get(url = 'https://api.cognitedata.com/api/0.5/projects/smart-maintenance-sandbox/timeseries/' + str(ts1_id) ,
                  headers= { 'Api-Key' : sm_api} , params = {"includeMetadata" : False})
print(r1.text.split('\n'))
# then with metadata
r1 = requests.get(url = 'https://api.cognitedata.com/api/0.5/projects/smart-maintenance-sandbox/timeseries/' + str(ts1_id) ,
                  headers= { 'Api-Key' : sm_api} , params = {"includeMetadata" : True})
print(r1.text.split('\n'))
```
**Expected behavior**
The `client.time_series.get_time_series(name = ts_name,include_metadata = True)` should return the metadata.

**Comment by jaketbouma** [comment]
This one's affecting me too.

**Comment by dshynkev** [comment]
I think the cause is that `to_pandas` has an independent `include_metadata` parameter which is false by default:
```python
class TimeSeriesListResponse(CogniteCollectionResponse):
    """Time series Response Object"""

    _RESPONSE_CLASS = TimeSeriesResponse

    def to_pandas(self, include_metadata: bool = False):
```

**Comment by erlendvollset** [comment]
#291 Fixes this. Will be released in 0.13.2 today.
